packages/next-on-pages/tests/_helpers/index.ts

```diff
@@ -309,3 +311,26 @@ export function createPrerenderedRoute(
 		[`${file}.rsc.prerender-fallback.rsc`]: `${fileWithBase}.rsc.prerender-fallback.rsc`,
 	};
 }
+
+type ConsoleMethods = {
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	[Method in keyof Console]: Console[Method] extends (...args: any[]) => any
```

Comment:
Ah, apologies. You can use `unknown` here instead.

```suggestion
	[Method in keyof Console]: Console[Method] extends (...args: unknown[]) => unknown
```

Reply:
I did change it to unknown, but it was missing quite a few of the options (e.g. `table` that you mentioned in your previous comment), so I changed it back to any and put the comment disabling eslint for it.

Reply:
Weird! Okay, whatever :)